modules/appeals_api/app/swagger/appealable_issues/v0/swagger.json

```diff
@@ -340,35 +340,45 @@
       },
       "productionOauth": {
         "type": "oauth2",
-        "description": "This API uses OAuth 2 with the authorization code grant flow. [More info](https://developer.va.gov/explore/authorization?api=claims)",
+        "description": "The authentication model for the Appealable Issues API uses OAuth 2.0/OpenID Connect. The following authorization models are supported: [Authorization code flow](https://developer.va.gov/explore/api/appealable-issues/authorization-code) and [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appealable-issues/client-credentials).",
```

Comment:
This wording was taken directly from the developer portal "Authorization and Access" description for each segmented Appeals API, example [here](https://staging-developer.va.gov/explore/api/higher-level-reviews/docs?version=current).